src/components/Navbar.jsx

```diff
@@ -1,7 +1,51 @@
-import React from "react";
+"use client";
 
-const Navbar = () => {
-  return <div>Navbar</div>;
+import React, { useState } from "react";
+import Image from "next/image";
+import Logo from "@/public/svg/J&MLogo.svg";
+import Navbar from "react-bootstrap/Navbar";
+import Nav from "react-bootstrap/Nav";
+import Link from "next/link";
+import { items } from "@/data/Nav";
+
+const Navigation = () => {
+  const [selected, setSelected] = useState("");
+
+  return (
+    <Navbar
+      collapseOnSelect
+      fixed="top"
```

Comment:
You might have these swapped and maybe that's why collapse doesn't work? 

IE: `expand="lg"
      fixed="top"`